qiskit/opflow/gradients/circuit_gradients/lin_comb.py

```diff
@@ -685,24 +699,17 @@ def _hessian_states(
         self,
         state_op: StateFn,
         meas_op: Optional[OperatorBase] = None,
-        target_params: Optional[
-            Union[
-                Tuple[ParameterExpression, ParameterExpression],
-                List[Tuple[ParameterExpression, ParameterExpression]],
-            ]
-        ] = None,
+        target_params=None,
```

Comment:
Could you (re-)add the type hint on this parameter?

Reply:
Sure ups.